server/lib/placeHelper.js

```diff
@@ -1,10 +1,31 @@
+export const shapeCategories = (defaultValue, params) => {
```

Comment:
what does this function do? and please recheck the order of your parameters. this will not handle all cases. tests will help